### PR TITLE
Create OrganizationBadge based on UserBadge

### PR DESCRIPTION
**Pull request by benediktvaldez**

### CODE REVIEW

**Comment by plouc** [comment]
@benediktvaldez, thanks, it's a good idea! Could you please update the README to provide some info on its usage ?

**Comment by benediktvaldez** [comment]
@plouc of course - how could I forget :)

**Comment by plouc** [comment]
@benediktvaldez, sorry, I missed it, but can you also add a test for the component (you can use [this test](https://github.com/plouc/mozaik-ext-github/blob/master/src/tests/UserBadge.test.js)  as a reference) ?

File: README.md [hunk 1]
```diff
@@ -199,10 +199,34 @@ key          | required | description
 }
 ```
 
+<<<<<<< Updated upstream
```
Comment (plouc) [inline README.md]: I think, you should resolve this conflict :)

**Comment by benediktvaldez** [comment]
@plouc missed the fact you had tests - fixed :smile: 

Also, rebased with your master so you should be able to merge safely

**Comment by plouc** [comment]
@benediktvaldez, just created a [demo dashboard for all available widgets](http://mozaik-github.herokuapp.com/), it fitted perfectly, but now I have to find a place for yours :)

**Comment by plouc** [comment]
@benediktvaldez, available in `mozaik-ext-github@1.1.0`

**Pull request merged**